backend/src/routes/units.ts

```diff
@@ -20,4 +20,7 @@ router.get("/", UnitController.getUnitsHandler);
 
 router.delete("/:id", requireHousingLocator, UnitController.deleteUnitsHandler);
 
+router.put("/:id", requireHousingLocator, UnitController.getUnitHandler);
+router.put("/:id/approve", requireHousingLocator, UnitController.getUnitHandler);
+
```

Comment:
Oops, this change looks like it's for your new task. Can we revert it and make the changes on a new branch?